core/domain/topic_domain.py

```diff
@@ -208,19 +208,20 @@ def validate(self):
                 % self.skill_ids)
 
     @classmethod
-    def create_default_topic(cls, topic_id):
+    def create_default_topic(cls, topic_id, name=feconf.DEFAULT_TOPIC_NAME):
```

Comment:
Ditto, don't need default value. In general try not to use default values where possible. Leads to more branching logic which is harder to maintain.

Reply:
Done.